evm/src/cpu/kernel/interpreter.rs

```diff
@@ -119,7 +125,7 @@ impl<'a> Interpreter<'a> {
             0x1b => todo!(),                                           // "SHL",
             0x1c => todo!(),                                           // "SHR",
             0x1d => todo!(),                                           // "SAR",
-            0x20 => todo!(),                                           // "KECCAK256",
+            0x20 => self.run_sha3(),                                   // "SHA3",
```

Comment:
nit: I think `KECCAK256` is the preferred name, the yellowpaper changed the mnemonic from `SHA3` at some point.